bluesky_queueserver/worker.py

```diff
@@ -22,12 +18,14 @@
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.DEBUG)
 
-config_bluesky_logging(level='INFO')
-config_ophyd_logging(level='INFO')
+logging.getLogger('bluesky').setLevel('WARNING')
```

Comment:
This is a useful workaround in scripts or interactive sessions when loggers get to noisy, but modules should not silence loggers at import time. See previous comment for a better approach.

Reply:
Controlling verbosity using -v is useful. I will add CLI options in one of the following PRs.